lib/game_logic.rb

```diff
@@ -1 +1,34 @@
-'hello'
+class Player
+  attr_accessor :name
+
+  def initialize(name)
+    @name = name
+  end
+end
+
+def greeting
+  puts 'TIC TAC TOE GAME'
+  puts 'TO START PRESS ENTER'
+  gets.chomp
+end
+
+def design_board(slots)
+  system('clear')
+  system('cls')
+
+  puts '+---+---+---+'
+  puts "| #{slots[0]} | #{slots[1]} | #{slots[2]} |"
+  puts '+---+---+---+'
+  puts "| #{slots[3]} | #{slots[4]} | #{slots[5]} |"
+  puts '+---+---+---+'
+  puts "| #{slots[6]} | #{slots[7]} | #{slots[8]} |"
+  puts '+---+---+---+'
+end
```

Comment:
No puts/print and gets.chomp outside main.rb file. Instead of using puts in the design_board function 
- [ ]  Create a return string variable
- [ ]  Assign to that variable the board and return